suite-native/navigation/src/components/Screen.tsx

```diff
@@ -115,19 +115,22 @@ export const Screen = ({
     isScrollable = true,
     hasStatusBar = true,
     backgroundColor = 'backgroundSurfaceElevation0',
-    customTopPadding = 'sp16',
-    customHorizontalPadding = 'sp16',
-    customBottomPadding = 'sp16',
+    noTopPadding = false,
+    noHorizontalPadding = false,
+    noBottomPadding = false,
     extraKeyboardAvoidingViewHeight = 0,
     hasBottomInset = true,
 }: ScreenProps) => {
     const {
         applyStyle,
-        utils: { colors, isDarkColor },
+        utils: { colors, isDarkColor, spacings },
     } = useNativeStyles();
 
-    const hasPaddingBottom = !useContext(BottomTabBarHeightContext) && hasBottomInset;
     const insets = useOfflineBannerAwareSafeAreaInsets();
+    const customTopPadding = noTopPadding ? 0 : spacings.sp16;
```

Comment:
super nit: I would maybe rename these values to not include `custom...` because it is not customizable anymore.

But feel free to resolve this.

Reply:
Fixed.